File: CocoaPod Samples/Sample /MainVC.m
/*
 * Licensed Materials - Property of IBM
 *
 * 5725E28, 5725I03
 *
 * © Copyright IBM Corp. 2011, 2015
 * US Government Users Restricted Rights - Use, duplication or disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
 */
#import "MainVC.h"
#import <IBMMobilePush/IBMMobilePush.h>

@interface MainVC ()

@end

@implementation MainVC

- (void)viewDidLoad {
    [super viewDidLoad];
    [version setText: MCESdk.sharedInstance.sdkVersion];
}

@end

File: samples/SwiftSample/MainVC.swift
/*
 * Licensed Materials - Property of IBM
 *
 * 5725E28, 5725I03
 *
 * © Copyright IBM Corp. 2015, 2016
 * US Government Users Restricted Rights - Use, duplication or disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
 */


import UIKit

class MainVC : UITableViewController
{
    @IBOutlet weak var version: UILabel?
    
    override func viewDidLoad() {
        super.viewDidLoad()
        version!.text = MCESdk.sharedInstance().sdkVersion()
    }
}
